src/xcode/ENA/ENA/Source/Workers/__tests__/LinkHelperTests.swift

```diff
@@ -0,0 +1,33 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import XCTest
+@testable import ENA
+
+final class LinkHelperTests: CWATestCase {
+	
+	func testFAQAnchorDE() throws {
+		let url = LinkHelper.urlString(suffix: "test", type: .faq)
+
+		XCTAssertEqual(NSLocale.current.languageCode, "de")
+		XCTAssertEqual(url, "https://www.coronawarn.app/de/faq/#test")
+	}
+	
+	func testFAQAnchorEN() throws {
+		if NSLocale.current.languageCode == "en" {
+			let url = LinkHelper.urlString(suffix: "test", type: .faq)
+
+			XCTAssertEqual(url, "https://www.coronawarn.app/en/faq/#test")
+		}
+	}
+	
+	func testFAQAnchorOtherLanguages() throws {
+		let supportedLocale = ["tr", "ro", "pl"]
+		if supportedLocale.contains(NSLocale.current.languageCode ?? "") {
+			let url = LinkHelper.urlString(suffix: "test", type: .faq)
+
+			XCTAssertEqual(url, "https://www.coronawarn.app/en/faq/#test")
+		}
+	}
```

Comment:
I think this structure will not work, as the first test would fail if the current `languageCode` is not "de". I think it will not fail for us because we only test unit tests in German. But with that in mind we will never test anything in the other two tests as the current language will always be German.

Idea one: Figure out a way to override the language settings in single tests.
Idea two: Don't use the `Localizable.links.strings` files, instead hardcode the URL in `urlString(suffix: String, type: LinkType)`, but change the signature to `urlString(suffix: String, type: LinkType, languageCode: String? = Locale.current.languageCode)` so you can pass it in for the tests and don't have to pass it in anywhere else.

Reply:
I already search if we have any mechanism to override locale for a particular unit test but unfortunately we don't have one, the unit tests are run with one locale. Have refactored it to option 2 which you have suggested.